miranda/convert/_reanalysis.py

```diff
@@ -0,0 +1,272 @@
+    target_chunks: dict, optional
+    output_format: {"netcdf", "zarr"}
+    overwrite: bool
+    engine: {"netcdf4", "h5netcdf"}
+    n_workers: int
+
+    Returns
+    -------
+    None
+    """
+    if output_format == "netcdf":
+        suffix = ".nc"
+    elif output_format == "zarr":
+        suffix = ".zarr"
+    else:
+        raise NotImplementedError(f"`output_format`: '{output_format}")
+
+    with ProgressBar(), dask.config.set(
+        **{"array.slicing.split_large_chunks": False},
+        n_workers=n_workers,
+        **dask_kwargs,
+    ):
+        out_files = Path(output_folder)
+        if isinstance(domains, str):
+            domains = [domains]
+
+        for domain in domains:
+            if domain == "_DEFAULT":
+                logging.warning("No domain specified. proceeding with 'not-specified'")
+                output_folder = output_folder
+                domain = "not-specified"
+            elif isinstance(domain, str):
+                output_folder = out_files.joinpath(domain)  # noqa
+            else:
+                raise NotImplementedError()
+
+            output_folder.mkdir(exist_ok=True)
+
+            for project, in_files in data.items():
+                logging.info(
+                    f"Processing {project} data{f' for domain {domain}' if domain !='not_specified' else ''}."
+                )
+                for var in variables:
+                    # Select only for variable of interest
+                    multi_files = sorted(x for x in in_files if f"{var}_" in str(x))
+
+                    if multi_files:
+                        all_chunks = get_chunks_on_disk(multi_files[0])
+                        chunks = all_chunks[var]
+
+                        if target_chunks is None:
+                            output_chunks = dict()
+                            mappings = dict(longitude="lon", latitude="lat")
+                            for k, v in chunks.items():
+                                if k in mappings.keys():
+                                    output_chunks[mappings[k]] = v
+                                else:
+                                    output_chunks[k] = v
+
+                            logging.warning(
+                                "No `target_chunks` set. "
+                                f"Proceeding with following found chunks: {output_chunks}"
+                            )
+                        else:
+                            output_chunks = target_chunks
+
+                        logging.info(f"Resampling variable `{var}`.")
+
+                        if aggregate:
+                            time_freq = aggregate
+                        else:
+                            parse_freq = calendar.parse_offset(
+                                xr.infer_freq(xr.open_dataset(multi_files[0]).time)
+                            )
+                            time_freq = f"{parse_freq[0]}{xarray_frequencies_to_cmip6like[parse_freq[1]]}"
```

Comment:
FIXME: This is writing out `11hr` for `1hr` data due to the parsing (scalar of `1` and units of `1hr`)

Reply:
This should be fixed with https://github.com/Ouranosinc/miranda/pull/24/commits/f2892562eda6b184e203da09e73596764a391a5a